userspace/ksud/src/cli.rs

```diff
@@ -307,7 +307,7 @@ pub fn run() -> Result<()> {
                 Module::Shrink => module::shrink_ksu_images(),
             }
         }
-        Commands::Install => utils::install(),
+        Commands::Install { magiskboot } => utils::install(magiskboot),
```

Comment:
You should add cli parameter here: https://github.com/tiann/KernelSU/pull/1684/files#diff-50c7b7af6d0fd53a343c78c5b80654b053d158d7e3b24e744040165af53cc505R38